src/projections/analysis/StsReader.java

```diff
@@ -121,7 +121,7 @@ private static String matchQuotes(StringTokenizer st) {
         StringBuilder value = new StringBuilder(current.substring(1));
         while (st.hasMoreTokens()) {
             current = st.nextToken();
-            value.append(current);
+            value.append(" " + current);
```

Comment:
Just to clarify, tokens are always separated by space (not by e.g. <tab>)?

Reply:
In the files we generate, yes. Even if other forms of whitespace snuck their way in, they'd still be tokenized correctly, just reduced to a single space when we reconstruct the representation here.